### PR TITLE
feat(TextInput/react): add autoCapitalize support for characters

### PR DESCRIPTION
**Pull request by haquezameer**

### CODE REVIEW

File: src/atoms/TextInput/utils.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const transformString = (string = '', transformType = 'none') => {
```
Comment (dhruvdutt) [inline src/atoms/TextInput/utils.js:1]: No need for util for this. You can directly do inside TextInput.web.js file.

File: src/atoms/TextInput/utils.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  switch (transformType) {
```
Comment (dhruvdutt) [inline src/atoms/TextInput/utils.js:2]: Add console warnings.

Comment (haquezameer) [reply src/atoms/TextInput/utils.js]: unsupported prop will throw error anyways.

File: src/atoms/TextArea/TextArea.stories.js [hunk 3]
```diff
@@ -13,6 +13,13 @@ const sizeOptions = {
   auto: 'auto',
 };
 
+const autoCapitalizeOptions = {
```
Comment (dhruvdutt) [inline src/atoms/TextArea/TextArea.stories.js:16]: Keep separate for web and mobile.

**Review by dhruvdutt — APPROVED** [review]

**Pull request merged**